modules/apps/object/object-service/src/main/java/com/liferay/object/internal/search/spi/model/index/contributor/ObjectEntryModelDocumentContributor.java

```diff
@@ -218,7 +226,22 @@ else if (value instanceof Integer) {
 		else if (value instanceof Long) {
 			_addField(fieldArray, objectFieldName, "value_long", valueString);
 
-			_appendToContent(sb, objectFieldName, valueString);
+			if (StringUtil.equals(
+					objectField.getBusinessType(),
+					ObjectFieldConstants.BUSINESS_TYPE_ATTACHMENT)) {
+
+				String fileName = _getFileName(GetterUtil.getLong(valueString));
+
+				_addField(
+					fieldArray, objectFieldName, "value_keyword_lowercase",
```

Comment:
Are we sure that we need the `value_keyword_lowercase` and, if so, since this only allows 256 chars, does it mean that we cannot search for keywords that are past that limit?

Reply:
No. The `value_keyword_lowercase`, as I understand it, is only used for sorting.
But now that you mention it, I remembered that it is necessary to truncate the text in `ObjectEntriesTableFDSView`.
I'll make this change. Thanks!

Reply:
I missed this message. I'm sorry.
I believe it's [used to search](https://github.com/liferay-forms/liferay-portal/pull/1471/files#diff-63928d3e39c7688c14b289e25efe811aeb96ee05a99f291ab9f754a96b35655bR132) as well, isn't it?

Reply:
Forget my question. It is. I just checked.